contents/spotlight/startup-11x.md

```diff
@@ -0,0 +1,90 @@
+
+At 11x, engineers rely on [session replay](/product/session-replay) to quickly solve customer issues instead of building complex error logging systems.
+
+"PostHog is like time travel," says Keith. "Otherwise we are banging our heads against the wall, spending days trying to debug."
+
+In one case, when a customer complained about unexpected behavior, an engineer reviewed the session replay and discovered the customer had changed their own app settings and forgotten about it.
+
+"We hadn't even thought about this," Keith explains. "We can just rewatch. All of our engineers use it now. It saves us building out error logs. We just rely on PostHog to be our timeline activity log."
+
+## Security monitoring
+
+Session replay even helped 11x identify and stop a security breach attempt. The team noticed 85% of their platform usage was coming from one small customer.
+
+"I could see they were trying to make unauthorized API calls to our platform," says Keith. "When we kicked them out, they threatened to sue us - but we were able to use our console logs to show their activity and prove they had violated our terms of service."
+
+(It should be noted that while Keith’s use is ingenious here, PostHog can’t legally guarantee that you, too, will be able to stop hackers at home.)
+
+## Consolidating tools, saving money
+
+Before standardizing on PostHog, 11x was using a mix of tools:
+- Custom-built feature flags
+- HotJar for session recordings
+- FullStory for product analytics
+
+"It was a hodgepodge of tools we were able to consolidate into one," Keith says. "And PostHog is the same price as each of them individually."
+
+## North Star metrics on display
+
+Keith's favorite use case is their office dashboard display. An automation runs daily to pull North Star metrics from PostHog to TV screens in their office.
+
+"Every day, it's so motivating to see that go up," he says, pointing to a rising graph of customer meetings booked. "The most exciting thing is, if it's not going up, I can do something about it."
+
+## What's next for 11x and PostHog
+
+While 11x is getting tremendous value from PostHog, Keith feels they're "barely using it." They haven't yet implemented [surveys](//surveys) or [A/B experiments](/experiments), and only recently discovered [LLM observability](/docs/ai-engineering/observability) - particularly relevant for an AI company.
+
+As 11x continues growing (they've relocated from London to San Francisco, raised $76 million, and now serve nearly 600 customers), PostHog remains central to their product-led approach.
+
+“PostHog is the ten-in-one product thing that helps us get so much done,” Keith declares. “You guys have it all.”
+
+## Build your own office dashboard
+
+Want to create an office metrics display like 11x? Here's their setup:
+
+**Ingredients:**
+- One Mac mini
+- One external display
+- One PostHog dashboard (11x uses their North Star metrics dashboard)
+- A script to refresh at your preferred interval
```

Comment:
This is a nice idea, but it's functionally useless unless we have more detail on that script and how it works. Otherwise it's just "Get a computer and set up a dashboard, so you can see the dashboard on the computer"

Reply:
@ivanagas could be a nice tutorial though?